src/components/CurrencyInput/CurrencyInput.spec.tsx

```diff
@@ -25,13 +25,13 @@ describe('CurrencyInput', () => {
    * Style tests.
    */
   it('should render with default styles', () => {
-    const actual = create(<CurrencyInput currency="EUR" />);
+    const actual = create(<CurrencyInput currency="EUR" label="Amount" />);
```

Comment:
This change is to fix deprecation warnings in jest, saying that labels will become required in the next major version. You can see the commit separately: 51035d3